c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -420,6 +426,7 @@ struct ExpandableSegment {
       CUmemGenericAllocationHandle handle = 0;
       CUmemAllocationProp prop = {};
       prop.type = CU_MEM_ALLOCATION_TYPE_PINNED;
+      prop.requestedHandleTypes = CU_MEM_HANDLE_TYPE_POSIX_FILE_DESCRIPTOR;
```

Comment:
I ask some Nvidia folks and they verified that this doesn't consume additional resources if we do not request a handle. So it is safe to do this for all expandable segments.

Reply:
For older drivers (like 525) this sometimes leads to big slowdowns.